### PR TITLE
Updated Sorting and Added Download to Excel Button

### PR DESCRIPTION
**Pull request by staradan**

### CODE REVIEW

File: src/components/quickViewTable/quickViewTable.js [hunk 1]
```diff
@@ -6,55 +6,27 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
...
-    //         statsObj.allNegative.push(stat);
-    //         statsObj[stat.playerName].negative.push(stat);
-    //     }
-    // });
-
-    // const [form, setState] = useState({
-    //     username: '',
-    //     password: ''
-    //   });
+    const { categories, stats, viewOnlyStats } = useContext(FirebaseContext);
```
Comment (pmiller19) [inline src/components/quickViewTable/quickViewTable.js:10]: nice deletion here

Comment (staradan) [reply src/components/quickViewTable/quickViewTable.js:10]: Thanks!

File: src/pages/home.js [hunk 2]
```diff
@@ -10,8 +10,47 @@ const stringifyDate = (date) => {
...
+                // This is to not add a comma at the last cell
+                // The '\r\n' adds a new line
+                if (key) {
+                    csv += key + (keysCounter + 1 < keysAmount ? ',' : '\r\n')
+                    keysCounter++
+                }
+            }
+        } else {
+            for (let key in data[row]) {
+                if (typeof data[row][key] === 'object') {
```
Comment (pmiller19) [inline src/pages/home.js:36]: GENIUS

**Review by pmiller19 — APPROVED** [review]
Good work Dan!

**Pull request merged**